juniper_official/routing/collect-isis-lsdb-lsp.rule

```diff
@@ -0,0 +1,98 @@
+                    path "/network-instances/network-instance/@name";
+                }			
+                type string;
+                description "instance-name from the lsdb path";
+            }
+            field protocol-name {
+                sensor isis-oc {
+                    where "/network-instances/network-instance/protocols/protocol/@identifier=~/{{proto-name}}/";
+                    path "/network-instances/network-instance/protocols/protocol/@identifier";
+                }			
+                type string;
+                description "protocol-name from the lsdb path";
+            }
+            field level-numb {
+                sensor isis-oc {
+                    where "/network-instances/network-instance/protocols/protocol/isis/levels/level/@level-number=~/{{level-num}}/";
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/@level-number";
+                }
+                type string;
+                description "level-number from the lsdb path";
+            }
+            field lsp-id {
+                sensor isis-oc {
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/@lsp-id";
+                }
+                type string;
+                description "lsp-id from the lsdb path";
+            }
+            field lsp-install-time {
+                sensor isis-oc {
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/state/lsp-install-time";
+                }
+                type string;
+                description "lsp-install-time from the lsdb path";
+            }
+            field sequence-number {
+                sensor isis-oc {
+                    path "/network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/lsp/state/sequence-number";
+                }
+                type string;
+                description "lsp-sequence-number from the lsdb path";
+            }
+            variable inst-name {
+                value .*;
+                description "Enter a regular expression to match the instance name, e.g.master ";
+                type string;
+            }
+            variable level-num {
+                value .*;
+                description "Enter a regular expression to match the level number, e.g., '1', '2', or '3' ";
+                type string;
+            }
+            variable proto-name {
+                value .*;
+                description "Enter a regular expression to match the protocol name, e.g., 'isis' or 'ospf' ";
+                type string;
+            }
+            rule-properties {
+                version 1;
+                contributor juniper;
+                supported-healthbot-version 4.3.0;
+                supported-devices {
+                    juniper {
+                        operating-system junos {
```

Comment:
Update all products and models supported by EOP

Reply:
Currently we are supporting only for MX960 -22.2R3 as min supported rel and will be adding other platforms during main testing phase.